deployment/stacks/parent.ts

```diff
@@ -50,7 +38,7 @@ export class ParentStack extends Stack {
       iamPath,
     };
 
-    const vpc = ec2.Vpc.fromLookup(this, "Vpc", { vpcName });
+    const vpc = ec2.Vpc.fromLookup(this, "Vpc", { vpcName: commonProps.vpcName });
```

Comment:
this is a nit, but why commonsProps and not props and why did we remove the deconstruction assigning const vpcName

Reply:
Fixed 👍